### PR TITLE
[IBD/Rewind] Package SC state and receipts in zip file

### PR DESCRIPTION
**Pull request by quantumagi**
https://app.clickup.com/t/20jbud6

Merge PR #926 first

### CODE REVIEW

**Comment by zeptin** [comment]
Just to be a pain, could you add a zip for CirrusTest as well? :D

File: src/Stratis.Bitcoin.Features.SmartContracts/SmartContractFeature.cs [hunk 1]
```diff
@@ -42,13 +46,40 @@ public sealed class SmartContractFeature : FullNodeFeature
...
+                {
+                    // Un-zip "contracts" and "contractsreceipts" to sub-folders within the data folder.
+                    if (Directory.Exists(this.dataFolder.SmartContractStatePath))
+                        Directory.Delete(this.dataFolder.SmartContractStatePath, true);
+
+                    if (Directory.Exists(this.dataFolder.SmartContractStatePath + "receipts"))
+                        Directory.Delete(this.dataFolder.SmartContractStatePath + "receipts", true);
+
+                    System.IO.Compression.ZipFile.ExtractToDirectory(checkPointFileName, this.dataFolder.RootPath);
+                }
```
Comment (zeptin) [inline src/Stratis.Bitcoin.Features.SmartContracts/SmartContractFeature.cs:81]: Hmm, shouldn't you `break` once the most recent zip is extracted? Obviously there would typically only be a single zip packaged with a release, but breaking seems safer.

Comment (quantumagi) [reply src/Stratis.Bitcoin.Features.SmartContracts/SmartContractFeature.cs:81]: Yep. Definitely missing a `break`. Good spot.

**Review by zeptin — APPROVED** [review]

**Comment by quantumagi** [comment]
This PR leverages the change in PR #923.

**Pull request merged**